MotionPhotoReader/app/src/main/java/com/google/android/libraries/motionphotoreader/MotionPhotoReader.java

```diff
@@ -95,49 +98,37 @@ private MotionPhotoReader(File file,
      * @param surface The surface for the motion photo reader to decode.
      * @param surfaceWidth The width of the surface, in pixels.
      * @param surfaceHeight The height of the surface, in pixels.
+     * @param stabilizationOn If true, the video will be stabilized
      * @return a MotionPhotoReader object for the specified file.
      * @throws IOException when the file cannot be found.
      * @throws XMPException when parsing invalid XML syntax.
      */
-    @RequiresApi(api = M)
     public static MotionPhotoReader open(File file,
                                          Surface surface,
                                          int surfaceWidth,
-                                         int surfaceHeight) throws IOException, XMPException {
+                                         int surfaceHeight,
+                                         boolean stabilizationOn
```

Comment:
I'm a bit confused what `stabilizationOn` means here. Can we rename to either:

1. `enableStabilization`   (or `applyStabilization`) -- if this value means we should apply stabilization
2. `isPrestabilized` -- if this value means the video is already stabilized  (not sure why we would need a isPrestabilized input, so I'm guessing it's item 1)